.drone.yml

```diff
@@ -228,7 +228,7 @@ steps:
         from_secret: passphrase
       commands:
         - cd /srv/lf/website/prod
-        - mv ./deploy/* ./
+        - rsync -r -I ./deploy/ ./ --delete-after
```

Comment:
If we do a --delete-after I think the next line is unnecessary?

Reply:
--delete-after works as follows: First synchronize all files, then looks which files are present at the source and which files are present at the destination, deletes all files found at the destination but not at the source.
So it does not delete the source directory (./deploy/), which we want I assume? Which is why the command after is necessary.

Reply:
TIL 🙏